### ISSUE TITLE
[🐛 BUG] Column headers of Taipy table are truncated

### ISSUE THREAD

**Issue by FlorianJacta** [Avaiga/taipy#2082]
### What went wrong? 🤔

When having a lot of columns in a table, the columns headers becomes truncated.

![image](https://github.com/user-attachments/assets/ae6c0184-a0ef-482d-9df6-69b1fdbba942)

This is a new behavior that didn't exist in 3.1.

![image](https://github.com/user-attachments/assets/572d772d-b8b1-4e1f-a55a-991c9b9a754c)

### Expected Behavior

We should see the entire name of the columns.

### Steps to Reproduce Issue

Run this code in 4.0:

```python
from taipy.gui import Gui
import taipy.gui.builder as tgb

data = {
    "a_long_title": [1],
    "b_long_title": [2],
    "c_long_title": [3],
    "d_long_title": [4],
    "e_long_title": [5],
    "f_long_title": [6],
    "g_long_title": [7],
    "h_long_title": [8],
    "i_long_title": [9],
    "j_long_title": [10],
    "k_long_title": [11],
    "l_long_title": [12],
    "m_long_title": [13],
    "n_long_title": [14],
    "o_long_title": [15],
    "p_long_title": [16],
    "q_long_title": [17],
    "r_long_title": [18],
    "s_long_title": [19],
    "t_long_title": [20],
    "u_long_title": [21],
    "v_long_title": [22],
    "w_long_title": [23],
}

with tgb.Page() as page:
    tgb.table("{data}")

Gui(page).run()

```

### Version of Taipy

4.0

### Acceptance Criteria

- [ ] Ensure new code is unit tested, and check code coverage is at least 90%.
- [ ] Create related issue in taipy-doc for documentation and Release Notes.

### Code of Conduct

- [X] I have checked the [existing issues](https://github.com/Avaiga/taipy/issues?q=is%3Aissue+).
- [ ] I am willing to work on this issue (optional)

**Comment by JeevaRamanathan** [comment]
![image](https://github.com/user-attachments/assets/e974b6b0-2a96-4357-8821-4002f4c4f81c)
Hi @FlorianJacta, is this issue open for working?

**Comment by FlorianJacta** [comment]
Our R&D is working on it @JeevaRamanathan

**Comment by AlexandreSajus** [comment]
Fixed in 4.0.1.dev1
🎉
